### DISCUSSION TITLE
Known Issues with pip install!

### DISCUSSION THREAD

**Discussion by raymondlo84** [openvinotoolkit/openvino_notebooks#26] [General]
I think many of us may have seen these warning messages during the pip install step! 

Please report what you see or broken if it is related to any of the following packages. We would love to hear from all of you on this as solving dependencies is definitely not an easy problem! 
```

ERROR: After October 2020 you may experience errors when installing or updating packages. This is because pip will change the way that it resolves dependency conflicts.

We recommend you use --use-feature=2020-resolver to test your packages with the new resolver before it becomes the default.

openvino-dev 2021.3.0 requires networkx==2.2, but you'll have networkx 2.5.1 which is incompatible.
openvino-dev 2021.3.0 requires numpy~=1.17.3, but you'll have numpy 1.18.5 which is incompatible.
tensorflow 2.4.1 requires numpy~=1.19.2, but you'll have numpy 1.18.5 which is incompatible.
jupyter-packaging 0.9.1 requires setuptools>=49.4.0, but you'll have setuptools 47.1.0 which is incompatible.
```

**Comment by joshiaru** [comment]
yes, i saw the ERROR text + networkx + numpy warnings when installing on a ubuntu18 image. Corrected with 
pip install numpy~=1.17.3 networkx==2.2

**Comment by Hobaspirit** [comment]
@raymondlo84 ,  Hello, I have the same issue : 
tensorflow  2.10.0 requires numpy>=1.20  and 
OpenVino requires numpy<1.20,>=1.16.6
Could you please help resolve this conflict?
Thanks,

**Comment by raymondlo84** [comment]
I always have to downgrade the tensorflow to match the system.

My best known method is really create a virtual environment (python -m venv openvino_env) and avoid these conflicts.
